tests/test_tlsfuzzer_analysis.py

```diff
@@ -17,6 +17,7 @@
     from tlsfuzzer.analysis import Analysis, main, TestPair, help_msg
     import pandas as pd
     import numpy as np
+    import multiprocessing as mp
```

Comment:
This is minor suggestion: mp is kind of similar to np mostly when comes to pronunciation. It may be a little bit confusing mostly when discussing the code. I would replace it with different abbreviation.

Reply:
hmm, true; on the other hand, it's used to get specific objects: Queue and Process for there, so it's unlikely to be very confusing